LDK/resources/web/LDK/plugin/UserEditableCombo.js

```diff
@@ -81,9 +81,11 @@ Ext4.define('LDK.plugin.UserEditableCombo', {
             },
 
             setValue: function(val){
-                this.addValueIfNeeded(val);
+                if (val) {
+                    this.addValueIfNeeded(val);
 
-                this.callOverridden(arguments);
+                    this.callOverridden(arguments);
+                }
```

Comment:
What's the rationale for the plugin dictating whether an empty value should or should not get passed to this.callOverridden(arguments)? Shouldnt that class determine how to deal with it?